doc/user/content/sql/drop-schema.md

```diff
@@ -26,20 +26,42 @@ Before you can drop a schema, you must [drop all sources](../drop-source) and
 
 ## Example
 
+### Remove a schema with no dependent objects
 ```sql
-SHOW SOURCES FROM my_schema
+SHOW SOURCES FROM my_schema;
 ```
 ```nofmt
 my_file_source
 ```
 ```sql
-DROP SCHEMA my_schema
+DROP SCHEMA my_schema;
+```
+
+### Remove a schema with dependent objects
+```sql
+SHOW SOURCES FROM my_schema;
 ```
 ```nofmt
-schema 'my_schema' cannot be dropped without CASCADE while it contains objects
+my_file_source
 ```
 ```sql
-DROP SCHEMA my_schema CASCADE
+DROP SCHEMA my_schema CASCADE;
+```
+
+### Remove a schema only if it has no dependent objects
+
+```sql
+DROP SCHEMA my_schema;
```

Comment:
We typically don't include the tautological outputs of commands, but we do include errors because the provide meaningful context that help users understand the examples.